tests/sqlparser_common.rs

```diff
@@ -995,6 +996,102 @@ fn parse_create_table() {
         .contains("Expected column option, found: GARBAGE"));
 }
 
+#[test]
+fn parse_create_table_if_not_exists() {
+    let sql = "CREATE TABLE IF NOT EXISTS uk_cities (\
+               name VARCHAR(100) NOT NULL,\
+               lat DOUBLE NULL,\
+               lng DOUBLE)";
+    let ast = one_statement_parses_to(
+        sql,
+        "CREATE TABLE IF NOT EXISTS uk_cities (\
+         name character varying(100) NOT NULL, \
+         lat double NULL, \
+         lng double)",
+    );
+    match ast {
+        Statement::CreateTable {
+            name,
+            columns,
+            constraints,
+            with_options,
+            if_not_exists: true,
+            external: false,
+            file_format: None,
+            location: None,
+        } => {
+            assert_eq!("uk_cities", name.to_string());
+            assert_eq!(
```

Comment:
Please remove the columns copy-paste, I feel the cost of duplication outweighs the benefit of making this test more realistic.